src/main/scala/org/dbpedia/spotlight/wikistats/ComputeStats.scala

```diff
@@ -86,33 +86,55 @@ class ComputeStats(lang: String) (implicit val sc: SparkContext,implicit val sql
           stemmer))
 
       textIds.map(textId => {
-        allOccFSASpotter.extract(textId._2)
-          .map(sfOffset => (textId._1,sfOffset._1))
+
+        var spots = ListBuffer[SurfaceFormOccurrence]()
+
+        textId._3.foreach(s => {
+
+          //Building the real Surface forms of the wiki article
+          val spotToAdd = new SurfaceFormOccurrence(new SurfaceForm(s._1),new Text(textId._2),s._2.toInt,Provenance.Annotation, -1)
```

Comment:
Consider adding some spaces between the arguments or else break down this line following scala's styleguide.